packages/zen-bull-netting/test/fuzzing/ZenBullNetting/DepositAuctionFuzzing.t.sol

```diff
@@ -84,11 +85,15 @@ contract DepositAuctionFuzzing is ZenBullNettingBaseSetup {
 
         uint256 crabAmount = _calAuctionCrabAmount(_amount);
         uint256 crabTotalSupply = IERC20(CRAB).totalSupply();
-        (, uint256 crabDebt) = IZenBullStrategy(ZEN_BULL).getCrabVaultDetails();
-        uint256 oSqthAmount = crabAmount * crabDebt / crabTotalSupply;
-
+        (uint256 crabCollateral, uint256 crabDebt) =
```

Comment:
Need the same in depositAuction integration tests (without angering the stack2deep gods)

Reply:
I don't understand this comment